frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.module.css

```diff
@@ -0,0 +1,12 @@
+.triggerButton {
+  border-radius: 0;
+  /** we hide the trigger button by default, and show it on hover */
+  visibility: hidden;
+  color: color-mix(in srgb, var(--mb-color-text-white), transparent 40%);
+  border-left: 2px solid
+    color-mix(in srgb, var(--mb-color-border), transparent 80%);
+}
+
+*:hover > .triggerButton {
```

Comment:
But I add a logic that shows the bucket button on hover here in the same file, so we don't need to add this logic to every call site.